src/libpsl-native/src/setdate.h

```diff
@@ -17,6 +17,6 @@ typedef struct setDateInfo
     int32_t DST;
 } SetDateInfo;
 
-int32_t SetDate(const SetDateInfo &info);
+int32_t SetDate(const SetDateInfo info);
 
```

Comment:
Why pass by value - the struct doesn't fit in registers.
If it's to be compatible with C, then maybe make it a pointer.